osnma-longan-nano-client/README.md

```diff
@@ -6,3 +6,16 @@ demo of the galileo-osnma library running on a small RISC-V GD32V103 microcontro
 
 The instructions for this demo are available in the
 [osnma-longan-nano README](https://github.com/daniestevez/galileo-osnma/tree/main/osnma-longan-nano)
+
+### Building
+
+The crate can be built using
+
+```
+cargo build --release
+```
+
+... and then run from the resulting path using the [demo instructions](https://github.com/daniestevez/galileo-osnma/tree/main/osnma-longan-nano):  
+```
+~/galileo-osnma/osnma-longan-nano-client/target/release/
```

Comment:
Just a nitpick. Maybe we want not to include `~/` here, since the user could have `galileo-osnma` anywhere. I think it's better to say just `galileo-osnma/osnma-longan-nano-client/target/release/`.